src/wrapper.test.js

```diff
@@ -68,56 +12,63 @@ const { version } = require('../package.json');
 describe('Distro initialization', () => {
   const ORIGINAL_PROCESS_ENV = process.env;
 
+  afterAll(() => {
+    process.env = ORIGINAL_PROCESS_ENV;
+  });
+
   beforeEach(() => {
-    process.env = { ...ORIGINAL_PROCESS_ENV };
+    /*
+     * We have a limit on the size of env we sent to the backend, and the env
+     * in the CI/CD goes over the limit, so the additional env vars we want to
+     * check for scrubbing get dropped.
```

Comment:
we can also set the env. var
OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT: "4096"

Reply:
Tracked in #153